lib/pages/new_group/widget/expansion_participants_list.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:fluffychat/pages/contacts/presentation/model/presentation_contact.dart';
+import 'package:fluffychat/pages/new_private_chat/widget/expansion_contact_list_tile.dart';
+import 'package:fluffychat/widgets/twake_components/twake_icon_button.dart';
+import 'package:flutter/material.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+
+class ExpansionParticipantsList extends StatefulWidget {
+
+  final Set<PresentationContact> contactsList;
+
+  const ExpansionParticipantsList({
+    super.key,
+    required this.contactsList,
+  });
+
+  @override
+  State<StatefulWidget> createState() => _ExpansionParticipantsListState();
+}
+
+class _ExpansionParticipantsListState extends State<ExpansionParticipantsList> {
+  bool isExpanded = true;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(left: 0.0, right: 0.0),
+      child: Column(
+        children: [
+          InkWell(
+            borderRadius: BorderRadius.circular(16.0),
+            onTap: () => toggleExpansionList(),
+            child: Padding(
+              padding: const EdgeInsets.only(left: 8.0, right: 2.0),
+              child: Row(
+                children: [
+                  SizedBox(
+                    height: 44,
+                    child: Row(
+                      children: [
+                        Text(L10n.of(context)!.participantsCount(widget.contactsList.length),
+                          style: Theme.of(context).textTheme.labelLarge?.copyWith(
+                            color: LinagoraRefColors.material().neutral[40],
+                          ),),
+                      ],
+                    ),
+                  ),
+                  const Expanded(child: SizedBox.shrink()),
```

Comment:
Isn't it the same utility as "Spacer()" ?

Reply:
no. it different.